### ISSUE TITLE
Ignores `sys` conditional compilation block

### ISSUE THREAD

**Issue by darmie** [HaxeFoundation/hxnodejs#185]
hxnodejs would not compile blocks that check if target is sys compatible.

e.g  the following code will not work on hxnodejs

```hx
#if sys 
Sys.println("Hello Haxe!")
#end
```

**Comment by neimanpinchas** [comment]
yes, because hxnodejs is not sync by default because it's not fully compatible

see this link https://github.com/HaxeFoundation/hxnodejs/blob/master/extraParams.hxml

https://community.haxe.org/t/using-sys-in-nodejs-target/3702/3
